.eslintrc.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     globalThis: false,
   },
   parser: '@typescript-eslint/parser',
-  plugins: ['@typescript-eslint', 'import', 'mocha'],
+  plugins: ['@typescript-eslint', 'import', 'mocha', 'simple-import-sort'],
```

Comment:
I have not look at `simple-import-sort`, `import` already has similar rule that has a fixer.